app/assets/stylesheets/frontend/views/_get-involved.scss

```diff
@@ -120,6 +99,10 @@
           padding: $gutter-half $gutter-half 0;
           a {
             color: $white;
+
+            &:focus {
```

Comment:
could you do `:link:focus` to avoid `important`?

Reply:
That gives a specificity of 1, 6, 2 - unfortunately the thing that this is trying to override is 1, 6, 3. So close! I've shuffled it down the page to nest it under `.attributes`, which then gives it 1, 8, 3.